Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Recurring_rare_domain_access_from_an_unsigned_process.yml

```diff
@@ -0,0 +1,933 @@
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "19"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": -553
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 7acd5fab-4226-408c-80ea-adeb6638ee6e
+    type: title
+    task:
+      id: 7acd5fab-4226-408c-80ea-adeb6638ee6e
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+      - "3"
+      - "34"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": -117
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 0547af53-c731-404f-8ad1-c44d730d29ad
+    type: regular
+    task:
+      id: 0547af53-c731-404f-8ad1-c44d730d29ad
+      version: -1
+      name: Get Process Hash Reputation
+      description: Retrieve results for a file hash using WildFire.
+      script: '|||file'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      file:
+        simple: ${alert.initiatorsha256}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 60,
+          "y": 9
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 69d4b951-f341-4662-8903-2d5981837170
+    type: regular
+    task:
+      id: 69d4b951-f341-4662-8903-2d5981837170
+      version: -1
+      name: Get Domain Reputation
+      description: Checks the reputation of a domain.
+      script: '|||domain'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      domain:
+        simple: ${Core.OriginalAlert.raw_abioc.event.domain}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 497.5,
+          "y": 9
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 5268de6d-14fa-4b08-9eee-3d893efeef78
+    type: condition
+    task:
+      id: 5268de6d-14fa-4b08-9eee-3d893efeef78
+      version: -1
+      name: Check for Malicious Reputation
```

Comment:
score > 2
WF verdict not benign
both can present suspicious - either change the name or the condition.